src/panoramic/cli/connections.py

```diff
@@ -235,6 +236,33 @@ def test(cls, credentials) -> Tuple[bool, str]:
         # Create dialect specific adapter that handles connections
         adapter_cls = get_adapter_class_by_name(credentials.type)
         adapter = adapter_cls(adapter_config)  # type: ignore
+        return adapter
+
+    @classmethod
+    def execute(cls, sql: str, credentials) -> Tuple[str, Any]:
+        adapter = cls.get_connection_adapter(credentials)
+
+        with adapter.connection_named('pano'):  # type: ignore
+            conn = adapter.connections.get_thread_connection()
+            with conn.handle.cursor():
+                try:
+                    res = adapter.execute(sql=sql, fetch=True)
+                    conn.handle.commit()
+                except Exception as e:
+                    if conn.handle and 'closed' in dir(conn.handle) and conn.handle.closed == 0:
+                        conn.handle.rollback()
+                    # FIXME: remove prints
+                    print(sql)
+                    print(e)
+                    raise
```

Comment:
noticed this bit: I'm gonna change this to log the sql on a debug logger and re-raise